releases/8.5/languages/ru.php

```diff
@@ -0,0 +1,79 @@
+
+    'no_discard_title' => 'Атрибут #[\NoDiscard]',
+    'no_discard_description' => '<p>Добавив атрибут <code>#[\NoDiscard]</code> к функции, PHP будет проверять, используется ли возвращаемое значение, и выдавать предупреждение, если это не так. Позволяет повысить безопасность API, где возвращаемое значение важно, но про него можно легко забыть.</p><p>Связанное приведение типов <code>(void)</code> может использоваться для указания, что значение намеренно не используется.</p>',
+
+    'persistent_curl_share_handles_title' => 'Постоянные дескрипторы cURL Share',
+    'persistent_curl_share_handles_description' => '<p>В отличие от <code>curl_share_init()</code>, дескрипторы, созданные с помощью <code>curl_share_init_persistent()</code>, не будут уничтожены в конце запроса PHP. Если найден постоянный дескриптор с тем же набором параметров, он будет использован повторно, что позволит избежать затрат на повторную инициализацию дескрипторов cURL при каждом запросе.',
+
+    'fcc_in_const_expr_title' => 'Замыкания и вызовы первого класса в константных выражениях',
+    'fcc_in_const_expr_description' => '<p>Статические замыкания и вызываемые объекты первого класса теперь могут использоваться в константных выражениях. Сюда входят параметры атрибутов, значения по умолчанию свойств и параметров, а также константы.</p>',
+
+    'new_classes_title' => 'Дополнительные функции и улучшения',
+    'fatal_error_backtrace' => 'Фатальные ошибки (такие как превышение максимального времени выполнения) теперь содержат обратную трассировку.',
+    'const_attribute_target' => 'Атрибуты теперь можно использовать с константами.',
+    'override_attr_properties' => 'Атрибут {0} теперь может использоваться со свойствами.',
+    'deprecated_traits_constants' => 'Атрибут {0} теперь может использоваться с трейтами и константами.',
+    'asymmetric_static_properties' => 'Статические свойства теперь поддерживают асимметричную видимость.',
+    'final_promoted_properties' => 'Свойства могут быть помечены окончательными (<code>final</code>) с помощью свойств в конструкторе.',
+    'closure_getCurrent' => 'Добавлен метод <code>Closure::getCurrent()</code> для упрощения рекурсии в анонимных функциях.',
+    'partitioned_cookies' => 'Функции {0} и {1} теперь поддерживают ключ <code>"partitioned"</code>.',
+    'get_set_error_handler' => 'Добавлены функции {0} и {1}.',
+    'new_dom_element_methods' => 'Добавлены методы {0} и {1}.',
+    'grapheme_levenshtein' => 'Добавлена фукнция {0}.',
+    'delayed_target_validation' => 'Добавлен атрибут {0}, который можно использовать для подавления ошибок компиляции атрибутов ядра и модулей, которые используются на недопустимых целях.',
+
+    'bc_title' => 'Устаревшая функциональность и изменения в обратной совместимости',
+    'bc_backtick_operator' => 'Гравис (<code>`</code>) как псевдоним для {0} больше не поддерживается.',
```

Comment:
```suggestion
    'bc_backtick_operator' => 'Обратный апостроф (<code>`</code>) как псевдоним для {0} больше не поддерживается.',
```

The gravis is a diacritical mark, while the backtick is a separate keyboard character.